handlers/challenge_handler.py

```diff
@@ -486,12 +556,14 @@ def __init__(self):
             "renamectf": CommandDesc(RenameCTFCommand, "Renames a ctf", ["old_ctf_name", "new_ctf_name"], None),
             "reload" : CommandDesc(ReloadCommand, "Reload ctf information from slack", None, None),
             "archivectf": CommandDesc(ArchiveCTFCommand, "Archive the challenges of a ctf", None, None, True)
```

Comment:
I made a mistake. There's a comma missing here, and there's a hanging comma after showcreds.